apps/sample-app/src/AuthApp.tsx

```diff
@@ -18,6 +18,7 @@ const AuthApp: React.FC = () => {
         await app.initialize();
         setIsInitialized(true);
       } catch (e) {
+        alert('initalization error catch');
```

Comment:
Nit: I think it might be more accurate from the user's perspective to see something like just 'initialization error'.